2021-05-09 00:11:23,662 INFO    MainThread:18991 [wandb_setup.py:_flush():69] setting env: {}
2021-05-09 00:11:23,662 INFO    MainThread:18991 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-09 00:11:23,662 INFO    MainThread:18991 [wandb_init.py:_log_setup():336] Logging user logs to /home/trenton-windows/repos/spicy-dann/wandb/run-20210509_001123-rshjx0mt/logs/debug.log
2021-05-09 00:11:23,662 INFO    MainThread:18991 [wandb_init.py:_log_setup():337] Logging internal logs to /home/trenton-windows/repos/spicy-dann/wandb/run-20210509_001123-rshjx0mt/logs/debug-internal.log
2021-05-09 00:11:23,663 INFO    MainThread:18991 [wandb_init.py:init():369] calling init triggers
2021-05-09 00:11:23,663 INFO    MainThread:18991 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-09 00:11:23,663 INFO    MainThread:18991 [wandb_init.py:init():418] starting backend
2021-05-09 00:11:23,663 INFO    MainThread:18991 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-09 00:11:23,697 INFO    MainThread:18991 [backend.py:ensure_launched():132] starting backend process...
2021-05-09 00:11:23,737 INFO    MainThread:18991 [backend.py:ensure_launched():137] started backend process with pid: 19027
2021-05-09 00:11:23,755 INFO    MainThread:18991 [wandb_init.py:init():423] backend started and connected
2021-05-09 00:11:23,855 INFO    MainThread:18991 [wandb_init.py:init():465] updated telemetry
2021-05-09 00:11:23,868 INFO    MainThread:18991 [wandb_init.py:init():484] communicating current version
2021-05-09 00:11:26,224 INFO    MainThread:18991 [wandb_init.py:init():489] got version response 
2021-05-09 00:11:26,224 INFO    MainThread:18991 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-09 00:11:26,453 INFO    MainThread:18991 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 00:11:31,519 INFO    MainThread:18991 [wandb_run.py:_console_start():1565] atexit reg
2021-05-09 00:11:31,528 INFO    MainThread:18991 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-09 00:11:31,530 INFO    MainThread:18991 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-09 00:11:31,646 INFO    MainThread:18991 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-09 00:11:31,647 INFO    MainThread:18991 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 00:11:31,649 INFO    MainThread:18991 [wandb_run.py:_config_callback():787] config_cb None None {'dataset': 'camelyon17', 'data_root': './WILDS', 'model_name': 'resnet18', 'batch_size': 16, 'n_epochs': 1, 'get_train_metrics': True, 'save_every': 5, 'run_name': 'dev-small', 'max_val_batches': 5}
2021-05-09 00:15:32,915 INFO    MainThread:18991 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-09 00:15:32,924 INFO    MainThread:18991 [wandb_run.py:_restore():1507] restore
